filters/include/pcl/filters/impl/convolution.hpp

```diff
@@ -54,17 +60,17 @@ pcl::filters::Convolution<PointIn, PointOut>::Convolution ()
 {}
 
 template <typename PointIn, typename PointOut> void
-pcl::filters::Convolution<PointIn, PointOut>::initCompute (PointCloud<PointOut>& output)
+Convolution<PointIn, PointOut>::initCompute (PointCloud<PointOut>& output)
 {
   if (borders_policy_ != BORDERS_POLICY_IGNORE &&
       borders_policy_ != BORDERS_POLICY_MIRROR &&
       borders_policy_ != BORDERS_POLICY_DUPLICATE)
     PCL_THROW_EXCEPTION (InitFailedException,
-                         "[pcl::filters::Convolution::initCompute] unknown borders policy.");
+                         "[Convolution::initCompute] unknown borders policy.");
```

Comment:
Replaced by mistake multiple places in this file

Reply:
Thanks!